src/modules/posts/posts.service.ts

```diff
@@ -0,0 +1,62 @@
+import { UsersService } from "modules/users/users.service";
+import { PostsRepository } from "./posts.repository";
+import { PostsCreateDto, PostsUpdateDto } from "./dto";
+import { AuthorizationError } from "shared/errors";
+import { BlogPost } from "shared/entities";
+import { v4 as uuid } from "uuid";
+
+export class PostsService {
+    static async createPost(
+        post: Omit<PostsCreateDto, "id">,
+        userId: string
+    ): Promise<BlogPost> {
+        await UsersService.getUserById(userId);
```

Comment:
Yes, I have global error handling that handles all the exceptions.
fixed, now assigning the result of call to a variable. I was using this call just to make sure that the user with this id exists (otherwise a not found error would be thrown).